roles/ceilometer-common/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: restart ceilometer services
+  service: name={{ item }} state=restarted must_exist=false
+  when: restart|default('True')
+  with_items:
+    - ceilometer-api
+    - ceilometer-collector
+    - ceilometer-agent-notification
+    - ceilometer-agent-central
+    - ceilometer-alarm-evaluator
+    - ceilometer-alarm-notifier
+    - ceiloemter-agent-compute
```

Comment:
It is fine to put the handlers in the common role but we will need to separate it into two restart service. We will need one with "restart ceilometer controller service", which will restart only the 6 controller processes and one with "restart ceilometer data service", which will restart the 1 data process. In each controller and data main.yml, it will call its own corresponding restart.  Right now, it will try to restart all of them in controllers and compute. I believe the all codes are in all 3 nodes, it will end up all 7 processes are running on all 3 nodes, I think.